lib/index.js

```diff
@@ -72,388 +35,421 @@ const defaultRules = [
   '._*',
   '**/._*/**',
   '*.orig',
-  '/package-lock.json',
-  '/yarn.lock',
-  '/pnpm-lock.yaml',
   '/archived-packages/**',
 ]
 
-// There may be others, but :?|<> are handled by node-tar
-const nameIsBadForWindows = file => /\*/.test(file)
-
-class Walker extends IgnoreWalker {
-  constructor (opt) {
-    opt = opt || {}
+const strictDefaults = [
+  // these are forcibly included at all levels
+  '!/readme{,.*[^~$]}',
+  '!/copying{,.*[^~$]}',
+  '!/license{,.*[^~$]}',
+  '!/licence{,.*[^~$]}',
```

Comment:
is this case-insensitive on a case-sensitive filesystem?

Reply:
yes, we pass `nocase` to minimatch from ignore-walk making these rules case insensitive everywhere